apps/qaqc/src/digital_ocean_utils.py

```diff
@@ -19,49 +15,6 @@
     f"https://edm-recipes.nyc3.cdn.digitaloceanspaces.com/datasets/{dataset}/{version}/config.json"
 )
 
-construct_branch_output_data_directory_url = lambda dataset, branch, version: (
-    f"https://edm-publishing.nyc3.digitaloceanspaces.com/{dataset}/{branch}/{version}/output"
-)
-
-construct_output_data_directory_url = lambda dataset, version: (
-    f"https://edm-publishing.nyc3.digitaloceanspaces.com/{dataset}/{version}/output"
-)
-
-load_dotenv()
-
-
-def get_datatset_config(dataset: str, version: str) -> dict:
-    response = requests.get(
-        construct_data_library_config_url(dataset=dataset, version=version), timeout=10
-    )
-    return json.loads(response.text)
-
-
-def get_latest_build_version(dataset: str) -> str:
-    # TODO handle lack of version file
-    version = requests.get(
-        f"{construct_output_data_directory_url(dataset=dataset, version='latest')}/version.txt",
-        timeout=10,
-    ).text
-    return version
-
-
-def get_data_library_sql_file(dataset: str, version: str) -> None:
-    sql_dump_file_path_s3 = construct_data_library_sql_url(dataset, version)
-    dataset_by_version = construct_dataset_by_version(dataset, version)
-    sql_dump_file_path_local = SQL_FILE_DIRECTORY / f"{dataset_by_version}.sql"
-
-    if not os.path.exists(SQL_FILE_DIRECTORY):
-        os.makedirs(SQL_FILE_DIRECTORY)
-
-    if os.path.exists(sql_dump_file_path_local):
-        print(f"sql dump file already pulled : ({sql_dump_file_path_s3}")
-    else:
-        print(f"getting sql dump file : {sql_dump_file_path_s3} ...")
-        data_mysqldump = requests.get(sql_dump_file_path_s3, timeout=300)
-        with open(sql_dump_file_path_local, "wb") as f:
-            f.write(data_mysqldump.content)
-
 
 # NOTE this class is a legacy approach used in many reports, but prefer a functional approach
 class DigitalOceanClient:
```

Comment:
So I was going to leave the last remnants of this for now - I had originally just pulled out things that could move over just sort of dropped in or were duplicated a bit with existing things, and that getting this out of here entirely might expand the scope too much... but looking at it now maybe it'd just be worth getting the last couple functions out of the app code entirely